VSCodium (1.81.1, undefined, desktop)
Jupyter Extension Version: 2023.7.1002162226.
Python Extension Version: 2023.14.0.
Platform: linux (x64).
Workspace folder ~/Documents/Cours/5A GPSE - MSc IoT/Mathématiques, Home = /home/mouss
08:08:39.801 [info] Telemetry level is off
08:08:39.801 [info] Experiments are disabled, only manually opted experiments are active.
08:08:40.286 [info] Start refreshing Interpreter Kernel Picker (1693980520286)
08:08:40.297 [info] Using Pylance
08:08:41.296 [info] Starting Kernel startUsingPythonInterpreter, .jvsc74a57bd0e7370f93d1d0cde622a1f8e1c04877d8463912d04d973331ad4851f04de6915a./bin/python./bin/python.-m#ipykernel_launcher  (Python Path: /bin/python, Unknown, 3.11.5) for '~/Documents/Cours/5A GPSE - MSc IoT/Mathématiques/Intro.ipynb' (disableUI=true)
08:08:41.433 [warn] Failed to get activated env vars for /bin/python in 514ms
08:08:41.436 [warn] Failed to get activated env vars for /bin/python in 112ms
08:08:41.450 [info] Process Execution: /bin/python -c "import site;print("USER_BASE_VALUE");print(site.USER_BASE);print("USER_BASE_VALUE");"
08:08:41.485 [info] Process Execution: /bin/python -m pip list
08:08:41.496 [info] Process Execution: /bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
08:08:41.509 [info] Process Execution: /bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9003 --control=9001 --hb=9000 --Session.signature_scheme="hmac-sha256" --Session.key=b"b4abde18-78ba-4121-aa6e-85e9e5760ac6" --shell=9002 --transport="tcp" --iopub=9004 --f=~/.local/share/jupyter/runtime/kernel-v2-1747G8X9AGOkD6Ys.json
    > cwd: ~/Documents/Cours/5A GPSE - MSc IoT/Mathématiques
08:08:42.116 [info] ipykernel version & path 6.23.1, ~/.local/lib/python3.11/site-packages/ipykernel/__init__.py for /bin/python
08:08:42.660 [info] Disposing HostJupyterExecution
08:08:42.660 [info] Shutting down notebooks for 54cf34ab-5be8-489a-9e5a-8e15316b0d64
08:08:42.661 [info] Dispose Kernel '~/Documents/Cours/5A GPSE - MSc IoT/Mathématiques/Intro.ipynb' associated with '~/Documents/Cours/5A GPSE - MSc IoT/Mathématiques/Intro.ipynb'
08:08:42.662 [info] Disposing HostJupyterExecution
08:08:42.676 [info] Dispose Kernel '~/Documents/Cours/5A GPSE - MSc IoT/Mathématiques/Intro.ipynb' associated with '~/Documents/Cours/5A GPSE - MSc IoT/Mathématiques/Intro.ipynb'
08:08:42.678 [info] Dispose Kernel process 1897.
08:08:42.705 [error] Kernel Launcher. launch, Class name = Ky, completed in 1381ms, has a falsy return value, Arg 1: ".jvsc74a57bd0e7370f93d1d0cde622a1f8e1c04877d8463912d04d973331ad4851f04de6915a./bin/python./bin/python.-m#ipykernel_launcher", Arg 2: 60000, Arg 3: "~/Documents/Cours/5A GPSE - MSc IoT/Mathématiques/Intro.ipynb", Arg 4: "~/Documents/Cours/5A GPSE - MSc IoT/Mathématiques", Arg 5: "" [Canceled: Canceled
	at e.throwIfCanceled (~/.vscode-oss/extensions/ms-toolsai.jupyter-2023.7.1002162226-linux-x64/out/extension.node.js:2:1745170)
	at Ky.launchProcess (~/.vscode-oss/extensions/ms-toolsai.jupyter-2023.7.1002162226-linux-x64/out/extension.node.js:24:101971)
	at async ~/.vscode-oss/extensions/ms-toolsai.jupyter-2023.7.1002162226-linux-x64/out/extension.node.js:24:100416]
